draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -117,16 +117,26 @@ wherever possible.
 
 ## Feature Summary
 
-* This specification defines
- - a format for the payload of an Issuer-signed JWT containing selectively disclosable claims that include object properties (name-value pairs), array elements, and nested data structures built from these,
- - a format for data associated with the JWT that enables selectively disclosing those claims,
- - facilities for binding the JWT to a key and associated data to prove possession thereof, and
- - a format, extending the JWS Compact Serialization, for the combined transport of the JWT and associated data that is suitable for both issuance and presentation.
-* An alternate format utilizing the JWS JSON Serialization is also specified.
-* This specification enables combining selectively disclosable claims with
-  clear-text claims that are always disclosed.
-* For selectively disclosable claims that are object properties, both the key and value are always blinded.
-
+This specification defines two primary data formats:
+
+1. SD-JWT is a composite structure issued by the issuer that consists of the following:
+  - A format for the payload of an Issuer-signed JWT containing selectively
+    disclosable claims that include object properties (name-value pairs), array
+    elements, and nested data structures built from these
+  - A format for data associated with the JWT that enables selectively
+    disclosing those claims
+  - A format extending the JWS Compact Serialization, allowing for the combined
+    transport of the Issuer-signed JWT and the disclosure data for selectively
+    disclosed claims
+  - An alternate format extending the JWS JSON Serialization, also allowing for
+    transport of the Issuer-signed JWT and disclosure data
```

Comment:
I'm aware that the proposed change is mostly re-using old text, but with the recent changes around JWS JSON-serialization, I propose to reword as follows:

```suggestion
  - A format for enabling selective disclosure in nested JSON data structures,
    supporting selectively disclosable object properties (name-value pairs) and array elements
  - A format for encoding the selectively disclosable data items
  - A format extending the JWS Compact Serialization, allowing for the combined
    transport of the Issuer-signed JSON data structure and the disclosable data items
  - An alternate format extending the JWS JSON Serialization, also allowing for
    transport of the Issuer-signed JSON data structure and disclosure data
```